ntc_templates/templates/alcatel_aos_show_system.textfsm

```diff
@@ -0,0 +1,27 @@
+Value DESCRIPTION (.+,.+,[^,]+)
+Value OBJECT_ID ([^,]+)
+Value UP_TIME (\d+ days \d+ hours \d+ minutes and \d+ seconds)
```

Comment:
```suggestion
Value UPTIME (.+)
```

Move `UP_TIME` to `UPTIME` as that is a common capture group name. (We're hoping to standardize/normalize where we can.)

Reply:
https://github.com/networktocode/ntc-templates/blob/master/docs/dev/data_model.md